andela/zabuza-cfh
# Chore/146623145/convert grunt to gulp

#146623145 Convert Grunt to Gulp

#### What does this PR do?
Change the task runner from grunt to gulp

#### Description of Task to be completed?
Helps automate the installation of project dependencies, running the server and running of tests.

#### How should this be manually tested?

- Clone the repository to your local computer
- Run npm install to save project dependencies
- Run mongo daemon in one of your terminal with the command 'mongod'
- Run 'npm start' in another terminal to start the server
- Also run 'npm test' to check tests statistics

#### Any background context you want to provide?
Common tasks  like watching file changes, concatenating/minifying files, prefixing files for different browsers, and linting Javascript can easily get more complicated as the codebase grows. Hence, using a dedicated task runner like Gulp allows for running build tasks using Javascript, easier organization, easier readability, and easier combining/batching of tasks

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
